.circleci/config.yml

```
@@ -459,6 +459,19 @@ jobs:
       FORCE_RELEASE: ON
       MAKEFLAGS: -j 10
 
+  b_ubu_static:
+    <<: *build_ubuntu2004
+    environment:
+      MAKEFLAGS: -j 10
+      CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DSOLC_LINK_STATIC=1 -DUSE_CVC4=OFF -DUSE_Z3=OFF
```

```suggestion
      CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DSOLC_LINK_STATIC=1 -DUSE_CVC4=OFF -DUSE_Z3=ON
```

:-)?
There's a static Z3 library in the docker image here, so it would work just fine, wouldn't it? And is another few megabytes really important for these builds? They're even less likely to be networked around a lot than the emscripten builds, aren't they?

---

Or is the problem that the static Z3 library depends on a dynamic ``libstdc++``? That might be... If so we can fix that (resp. provide a static z3 library that can be used for an entirely static link in the docker image), and this of course doesn't need to be in this PR.